src/scripts/configure.ac

```diff
@@ -1441,6 +1441,20 @@ test_linkc "whether synchronization primitives (__sync_synchronize) works in C"
 AC_DEFINE_UNQUOTED(CMK_C_SYNC_SYNCHRONIZE_PRIMITIVE, $pass, [whether sync_synchronize primitives works in C])
 
 
+### test for _IO_file_overflow ###
```

Comment:
Is this test also include in the cmake build system?

Reply:
It was in this commit: https://github.com/UIUC-PPL/charm/pull/2531/commits/2121c83aa4f1a6996f70bcdd7355d6ff2cff258d It looks like it needs to be re-added.

Reply:
Sorry, I accidentally removed it when I was resolving a merge conflict. I've added it back in now in d8f8cfd.